roles/misc-perftools/tasks/main.yml

```diff
@@ -13,6 +13,20 @@
     - iotop
     - bwm-ng
 
+- name: Create directory for kvm metric collection configuration file
+  file: path=/opt/telegraf state=directory mode=0755
+  when: "'compute_hosts' in group_names"
+
+- name: Set configuration file for kvm metric collection
+  template:
+    src: /root/os-ansible-perftools/roles/misc-perftools/templates/kvm_virsh.py
```

Comment:
This won't work. Don't use the template module, there's a file module. You need to put the file under the `files/` directory for this role, and just use the basename `kvm_virsh.py`. 

I'm not sure if you have to prefix the name with 'files/' or not. Here's Kevin's example.
https://github.com/openstack/openstack-ansible-ops/blob/2a1e702209cc713861bae54be2324cd2a59b3121/cluster_metrics/playbook-influx-telegraf.yml#L38-L44